### ISSUE TITLE
Automatic registration of a twitter account.

### ISSUE THREAD

**Issue by Robijntjee** [N3MTV/gcphone#286]
Hello!

I use this fantastic phone, also with the twitter app, what strikes me is that you have to register a twitter account, now I have many people on our server who abuse it, create accounts on other people's names. So that the other gets a punishment that he himself has not done.

So I would like to delete the option to register an account, your twittername is your character name and you have no password, so you also have no login page for the twitter app.

**In other words:
-** Removing the login page and automatically creating a twitter account as soon as a person joins the server for the first time.

Greetings.

**Comment by ZeProf2Code** [comment]
or you can just verify steam identifier of the account in the DB :)

**Comment by Supremeeeeeee** [comment]
did you success to this wouldnt mind figuring this out also
